fibonacci/main.go

```diff
@@ -3,12 +3,29 @@ package main
 import "fmt"
 
 func main() {
-	fmt.Println(fibonacci(7))
+	const n = 5
+	fmt.Println("Calculating %d sequence\n", n)
 }
 
-func fibonacci(n int) int {
-	// Fill me in!
-	// When n is 0 or 1, fibonacci(n) should just be n.
-	// When n is any larger number,
-	// fibonacci(n) should be the sum of the previous two fibonacci numbers.
+func fibonacci(n int, old, new, calc string) int {
```

Comment:
Don't change this line.

When you think about it abstractly (never mind how to translate it to computer code), a Fibonacci calculation needs one piece of input - which Fibonacci number you're calculating - and produces one piece of output. That's what (the original version) of this line says: "take an integer as input and call it `n`; produce an integer as output."

The extra arguments you've added here seem to be inspired by the Towers of Hanoi example, but they don't apply. For that problem, there are four pieces of input - the number of discs, and the names of the "from," "to," and "spare" pins.